File: codeforces/low_difficulty/difficulty_1000/2.string_task_[118A]/string_task.py
# Author: José Rodolfo (jric2002)
s = str(input())
vowels = ['a', 'o', 'y', 'e', 'u', 'i']
o = ""
for ch in s:
  is_vowel = False
  ch = ch.lower()
  for v in vowels:
    if (v == ch):
      is_vowel = True
      break
  if (not(is_vowel)):
    o += '.' + ch
print(o)

File: codeforces/low_difficulty/difficulty_1000/2.string_task_[118A]/StringTask.java
/* Author: José Rodolfo (jric2002) */
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class StringTask {
  public static void main(String[] args) throws IOException {
    Character[] vowels = new Character[]{'a', 'o', 'y', 'e', 'u', 'i'};
    String s, o;
    Boolean is_vowel;
    BufferedReader input = new BufferedReader(new InputStreamReader(System.in));
    s = input.readLine();
    o = "";
    for (Short i = 0; i < s.length(); i++) {
      is_vowel = false;
      for (char v : vowels) {
        if (v == Character.toLowerCase(s.charAt(i))) {
          is_vowel = true;
          break;
        }
      }
      if (!is_vowel) {
        o += "." + Character.toLowerCase(s.charAt(i));
      }
    }
    System.out.println(o);
  }
}